helper/ssh/key_pair.go

```diff
@@ -0,0 +1,268 @@
+package ssh
+
+import (
+	"bytes"
+	"crypto/dsa"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"strings"
+
+	"golang.org/x/crypto/ed25519"
+	gossh "golang.org/x/crypto/ssh"
+)
+
+const (
+	// That's a lot of bits.
```

Comment:
BTW, is 4096 actually too many bits? The comment was a bit of a joke :)

Reply:
Oh yes, I meant to be more explicit here, I think it would be better to have something more informative as a comment in here. But I still think this is optional.

Reply:
I added some more detail to the comment.